src/reconcile/rec_hs.c

```diff
@@ -347,8 +349,14 @@ __rec_hs_cursor_pos(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor, uint32_t btr
      * If we find a key with a timestamp larger than or equal to the specified timestamp then the
      * specified timestamp must be mixed mode.
      */
-    WT_ASSERT_ALWAYS(
-      session, ts == 1 || ts == WT_TS_NONE, "out-of-order timestamp update detected");
+    char tp_string[4][WT_TS_INT_STRING_SIZE];
```

Comment:
What does tp mean? Is it time point?

Reply:
yes it's time point. I can rename to ts_string for consistency

Reply:
time point is the combination of txn id, start ts, and durable ts. Here we are printing a single ts. ts_string should be better.